leopardslab/nodecloud-legacy
# Implement a simple goto website

## What did you implement:

Closes #34 

## How did you implement it:

- Implemented a simple goto website using HTML5 and Bootstrap 3 framework.

## How can we verify it:

Visit the following link to see a preview of the website,
http://nc-front.s3-website.ap-south-1.amazonaws.com

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
